.github/actions/docker/container/is-healthy/action.yml

```diff
@@ -15,6 +15,11 @@ inputs:
     required: true
     default: 1
 
+  debug:
+    type: boolean
+    default: false
```

Comment:
This could be applied to other checks for simplification: `is-running`, `is-reachable`?